reference/commands/source.rst

```diff
@@ -0,0 +1,25 @@
+.. _reference_commands_source:
+
+conan source
+============
+
+.. code-block:: bash
+
+    $ conan source --help
+    usage: conan source [-h] [-v [V]] [--logger] [--name NAME] [--version VERSION] [--user USER] [--channel CHANNEL] [path]
+
+    Calls the source() method
+
+    positional arguments:
+    path               Path to a folder containing a recipe (conanfile.py or conanfile.txt) or to a recipe file. e.g.,
+                        ./my_project/conanfile.txt.
```

Comment:
This is just copy&paste from the --help of the commands and we will check all of them in the conan repo in case something needs changes but they can't be addressed in this one.